web/src/data/constants.ts

```diff
@@ -12,7 +12,7 @@ export const selectableCitiesData: Array<City> = [
   },
   {
     id: 120658,
-    name: "New York City",
```

Comment:
Seems the city name changed in the database. The coordinate city name is now only city name